pkg/auth/authproviders/openshift/watcher.go

```diff
@@ -0,0 +1,108 @@
+package openshift
+
+import (
+	"context"
+	"os"
+	"path"
+	"time"
+
+	"github.com/stackrox/rox/pkg/k8scfgwatch"
+	"github.com/stackrox/rox/pkg/logging"
+	"go.uber.org/zap"
+)
+
+var (
+	log = logging.LoggerForModule()
+
+	_ k8scfgwatch.Handler = (*handler)(nil)
+)
+
+type notifyOnCertPoolUpdate = func()
+
+func watchCertPool(n notifyOnCertPoolUpdate) {
+	opts := k8scfgwatch.Options{
+		Interval: 5 * time.Second,
+		Force:    true,
+	}
+
+	_ = k8scfgwatch.WatchConfigMountDir(context.Background(), path.Dir(serviceOperatorCAPath),
+		k8scfgwatch.DeduplicateWatchErrors(&handler{readCAs: internalCAs, notifyBackend: n}), opts)
```

Comment:
Word `notify` does not sit right with me. Perhaps `onCertPoolUpdate` - here and in `notifyOnCertPoolUpdate`?